src/raspbian/12/helix/arm32v6/Dockerfile

```diff
@@ -45,8 +45,10 @@ RUN apt-get update && \
         libunwind8 \
         locales \
         locales-all \
+        pkg-config \
         python3-dev \
         python3-pip \
+        rustc \
```

Comment:
https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1145/files installs rust from rustup instead because apparently the rustc in bookworm is too old?